lib/rails_util/timezone_helper.rb

```diff
@@ -14,42 +14,40 @@ class << self
       # @param [Symbol=>[Date]] options key-value option pairs, used to provide a separate Date object if a specific date is needed, otherwise the date of the time argument is used
       # @return DateTime object with the time in the desired `to_timezone`, and a date of either the provided date or time
       def convert_timezone(time, to_timezone, from_timezone, **options)
-        date = options.fetch(:date, time)
+        date = options[:date]
```

Comment:
To provide a more helpful exception message, should be keep `options.fetch` but remove the default option of `time`?

Reply:
I'm a bit turned around on the logic here. Could the code read:
```
offset_in_seconds = Time.zone.now.in_time_zone(to_timezone).utc_offset - Time.zone.now.in_time_zone(from_timezone).utc_offset
new_datetime = DateTime.new(date.year, date.month, date.day, time.hour, time.min, time.sec, format_offset(to_timezone)) + offset_in_seconds.seconds
```

Reply:
Perhaps we use `Time.now.in_time_zone...` instead [reference](https://stackoverflow.com/a/25143932).

Reply:
I think we want the time provided! (`time.in_time_zone(timezone)`)